code/development-env/docker/README.md

```diff
@@ -1,11 +1,24 @@
 # Setup local development environment using docker-compose
 
+## Pre-requisites
+
+* [Install Homebrew](https://brew.sh/)
+* [Install Docker for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac).
+* [Install kubectl with Homebrew](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-macos)
+* Use official kubectl, not Docker's: `brew link --overwrite kubernetes-cli`
+* [Install VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+* [Set Oracle security](https://stackoverflow.com/questions/52277019/how-to-fix-vm-issue-with-minikube-start)
```

Comment:
The name `Set Oracle security` doesn't seem quite right. Was this because the package is unsigned? I just upgraded to v6 but didn't see this issue... I guess I resolved it a long time ago!

Reply:
You probably only have to do it once.  I'll change the name to "Allow system software from Oracle".

Reply:
Great, thanks